packages/backend-defaults/api-report-rootHttpRouter.md

```diff
@@ -111,6 +113,15 @@ export interface MiddlewareFactoryOptions {
   logger: LoggerService;
 }
 
+// @public
+export class RateLimitStoreFactory {
```

Comment:
Do we need to make this public? Maybe best to hide it until we know that we need to externalize